.gitignore

```diff
@@ -0,0 +1 @@
+doc/tags
```

Comment:
This project lacks a proper `.gitignore` file, this template can be a good start: https://github.com/github/gitignore/blob/master/Global/Vim.gitignore